backends/dpdk/options.cpp

```diff
@@ -8,11 +8,18 @@ namespace P4::DPDK {
 using namespace P4::literals;
 
 std::vector<const char *> *DpdkOptions::process(int argc, char *const argv[]) {
+    auto executablePath = getExecutablePath(argv[0]);
+    if (executablePath.empty()) {
+        std::cerr << "Could not determine executable path" << std::endl;
+        return nullptr;
+    }
+    exe_name = cstring(executablePath.stem().c_str());
```

Comment:
Seems to be unused, no?

Reply:
This sets the variable here, which is used by other parts in the compiler.

https://github.com/p4lang/p4c/blob/main/frontends/common/parser_options.h#L75

Imho should be set whenever we call `getExecutablePath`

Reply:
Ah, ok, very.. unexpected side effect. Can we spell it like `this->exe_name` to outline that this is member variable?